playbooks/common/openshift-master/additional_config.yml

```diff
@@ -19,6 +19,7 @@
   roles:
   - role: openshift_master_cluster
     when: openshift_master_ha | bool and openshift.master.cluster_method == "pacemaker"
+  - role: openshift_project_request_template
```

Comment:
I would prefer to see the bool applied here.

Reply:
Done, also in `playbooks/common/openshift-cluster/upgrades/post_control_plane.yml`.

Reply:
Thanks for updating this.